### PR TITLE
Contact table in about page made responsive

### LINKED ISSUE #9461: About Page Contacts List Not Responsive
image

### PR DESCRIPTION
**Pull request by Trion129**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue: Closes #9461
#### Description

Made the table responsive by making the data elements stack when screen is small.

### CODE REVIEW

File: server/views/partials/stylesheets.jade [hunk 1]
```diff
@@ -3,3 +3,11 @@ link(rel="stylesheet" type="text/css" href="/bower_components/cal-heatmap/cal-he
 link(rel='stylesheet', href='/bower_components/font-awesome/css/font-awesome.min.css')
 link(rel='stylesheet', href='/bower_components/lightbox2/dist/css/lightbox.css')
 link(rel='stylesheet', href=rev('/css', 'main.css'))
+style.
```
Comment (Bouncey) [inline server/views/partials/stylesheets.jade]: This should be in `main.less`

**Comment by Bouncey** [comment]
Looks good, just need to put the styles in the right place :+1:

**Comment by raisedadead** [comment]
@Trion129 Thanks for the PR would you mind rebasing this off staging, and clear the conflict?
Hit us up on the chat if you need assistance?

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @Trion129 !

**Pull request merged**